src/webgpu/shader/validation/decl/var_access_mode.spec.ts

```diff
@@ -0,0 +1,226 @@
+export const description = `
+7.3 var Declarations
```

Comment:
nit: I don't think we reference the spec so tightly in the description anywhere else. The spec has a tendency of changing, and having stale text has been a problem before in the past. Maybe its more stable now?

We do have the `.specURL()` decoration that you can pass to the test, if that helps.